client/src/containers/HomeContainer.tsx

```diff
@@ -13,8 +12,6 @@ function mapStateToProps(state: State): HomeProps {
 }
 
 function mapDispatchToProps(dispatch: any): {} {
-    // Note, loads lesson on every vist of Home
-    dispatch(loadCourses());
```

Comment:
Are courses no longer going to be on the homepage? If so, what are we going to have instead?